icons/docsify.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Docsify</title><path d="M12 2.86c-6.622 0-12 5.377-12 12 0 1.659.355 2.918 1.025 3.852.669.933 1.64 1.513 2.776 1.864 2.274.7 5.252.55 8.447.55 3.194 0 6.05.153 8.205-.552 1.078-.353 1.99-.94 2.609-1.874.618-.934.938-2.186.938-3.84 0-6.623-5.377-12-12-12zm0 .787c6.198 0 11.214 5.016 11.214 11.213 0 1.552-.299 2.636-.807 3.405-.51.767-1.237 1.247-2.197 1.561-1.92.628-4.745.513-7.961.513s-6.164.118-8.215-.515c-1.026-.316-1.817-.803-2.369-1.571-.55-.769-.877-1.846-.877-3.393C.785 8.666 5.806 3.644 12 3.647zM7.8 5.535a.24.24 0 0 0-.118.023s-.758.374-1.4.826c-.644.454-.642.468-.892.694-.696.627-1.807 2.186-1.807 2.186a.251.251 0 0 0 .02.303l.26.29c.075.082.181.074.241-.017 0 0 .461-.725 1.01-1.36.55-.639.347-.406.742-.828.582-.62 2.354-1.51 2.354-1.51.095-.048.115-.152.04-.235l-.26-.29a.271.271 0 0 0-.189-.082zm.267 3.709a1.377 1.377 0 1 0 0 2.754 1.377 1.377 0 0 0 0-2.754zm8.656 0a1.377 1.377 0 1 0-.002 2.754 1.377 1.377 0 0 0 .002-2.754zm-13.575.983a.394.394 0 1 0 .002.788.394.394 0 0 0-.002-.788zm11.547 3.739c-.603-.01-1.482.287-2.493.287-1.01 0-1.948-.296-2.604-.287-.58.007-.941.25-.941 1.14.05 4.566 6.765 4.6 6.885 0 0-.89-.316-1.134-.847-1.14z"/></svg>
```

Comment:
Sorry for the delay!

A lot of improvements have being made so far! Ex.
![image](https://user-images.githubusercontent.com/12817388/143466693-4cd1f01e-5688-42c3-adc6-17ddc3bdab68.png)

There are still small differences around the mouth. You probably need to use a precision of 4 while optimizing the icon.